eye_on_hand: false
robot_base_frame: workspace_link
tracking_base_frame: camera3_link
transformation: {x: -0.228950876383, y: 0.462922605191, z: 0.903990165701, qx: 0.455912274876, qy: 0.354553466324, qz: -0.561549973132, qw: 0.592534827767 }
